et_ee.lang

```diff
@@ -0,0 +1,402 @@
+gui.xaero_subworld_dimension=Alam-maailm/dimensioon
+gui.xaero_switch_waypoint_set=Lülita teepunkti kogumit
+gui.xaero_waypoint_opacity_ingame=Teepunkti mängusisene läbipaistmatus
+gui.xaero_waypoint_opacity_map=Teepunkti kaardipealne läbipaistmatus
+gui.xaero_hide_world_names=Peida IPd/maailma nimed
+gui.xaero_open_slime=Ava klahvivajutusel
+gui.xaero_options=Valikud
+gui.xaero_make_automatic=Tee maailm/server autom.
+gui.xaero_make_automatic_msg1=Kas oled kindel?
+gui.xaero_make_automatic_msg2=See vahetab kõigi valitud maailma/serveri teepunktide andmed automaatse vastu, simuleerides valitud maailma/serveri automaatseks tegemist. Veendu, et tead, mida teed.
+gui.xaero_make_multi_automatic=Tee alam-maailm autom.
+gui.xaero_make_multi_automatic_msg1=Kas oled kindel?
+gui.xaero_make_multi_automatic_msg2=See vahetab kõigi valitud alam-maailma teepunktide andmed automaatse vastu, simuleerides valitud alam-maailma automaatseks tegemist. Veendu, et tead, mida teed.
+gui.xaero_delete_world=Kustuta maailm/server
+gui.xaero_delete_world_msg1=Kas soovid kustutada kõik teepunktide andmed valitud maailma/serveri jaoks?
+gui.xaero_delete_world_msg2=Seda tegevust ei saa tagasi võtta! Veendu, et tead, mida teed.
+gui.xaero_delete_multi_world=Kustuta alam-maailm
+gui.xaero_delete_multi_world_msg1=Kas soovid kustutada kõik teepunktide andmed valitud alam-maailma jaoks?
+gui.xaero_delete_multi_world_msg2=Seda tegevust ei saa tagasi võtta! Veendu, et tead, mida teed.
+gui.xaero_always_show_distance=Alati kauguse teepunktini kuvamine
+gui.xaero_item_tooltip=Eseme hüpiktekst
+gui.xaero_entity_tooltip_settings=Eseme hüpikteksti seaded
+gui.xaero_item_tooltip_min_lines=Minimaalselt ridu
+gui.xaero_item_tooltip_time=Kuvamisaeg
+gui.xaero_multiply_all_by_8=Alam-maailma koordinaadid * 8
+gui.xaero_divide_all_by_8=Alam-maailma koordinaadid / 8
+gui.xaero_multiply_msg1=Seda valikut peaks kasutama vaid ebakorrektsete teepunkti koordinaatide parandamiseks.
+gui.xaero_multiply_msg2=Kas oled kindel, et soovid kõik alam-maailma koordinaadid KORRUTADA 8-ga?
+gui.xaero_divide_msg2=Kas oled kindel, et soovid kõik alam-maailma koordinaadid JAGADA 8-ga?
+gui.xaero_share=Jaga
+gui.xaero_share_msg1=Kas soovid kindlasti jagada seda teepunkti vestluses §cKÕIGIGA§f?
+gui.xaero_share_msg2=Veendu, et sa ei paljasta salajast asukohta!
+gui.xaero_roof=lagi
+gui.xaero_partial=osaliselt
+gui.xaero_full=täielikult
+gui.xaero_show_light_level=Valgustaseme kuvamine
+gui.xaero_up=[Mine üles]
+gui.xaero_down=[Mine alla]
+gui.xaero_display_all_sets=Lülita kõigi teepunktide kogumite nähtavus
+gui.xaero_waypoint_name=teepunkti nimi
+gui.xaero_set_name=Määra nimi
+gui.xaero_render_layer=Ülekatte kuvamise kiht
+gui.xaero_visit_needed=§eUude vormingusse konvertimiseks pead enne seda dimensiooni külastama!
+gui.xaero_mp_cross_tp=Mitmikmängu mitme dimensiooni telep.
+gui.xaero_cross_tp=Mitme dimensiooni telep.
+gui.xaero_unreachable_dimension=Siht-dimensioon pole ligipääsetav.
```

Comment:
Literally "target dimension is not accessible".

Reply:
That sounds fine but it could be better. Usually (but not always) the dimension is "unreachable" when it can't even be found for a provided key/ID. I wouldn't use the word "accessible" in that context, but it would technically be not accessible, I guess. You can decide if you want to change it.